### PR TITLE
Refactor project directory structure

### LINKED ISSUE #10: Refactor project structure

### PR DESCRIPTION
**Pull request by Vulfpeck**
Fixes #10

### CODE REVIEW

**Review by AJaxRUN — APPROVED** [review]

File: src/Components/Common/AuthorInfo/AuthorInfo.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const AuthorInfo = () => {
+    return (
+        <div className="flex">
+            <img src="../assets/icons/image 1.png" className="mt-1 w-8 h-8 "/>
+            <div className="pl-3">
+                <p className="text-base font-semibold">Carl Johnson</p>
+                <p className="leading-tight text-sm text-nebula-grey-600">Risk
+                  Analysis</p>
+                {/* <p className="leading-tight text-s ">created on 17/01/2020</p> */}
```
Comment (sanghaiadarsh) [inline src/Components/Common/AuthorInfo/AuthorInfo.js:11]: do we need the commented bit here for future use?

File: src/Components/Common/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const Dropdown = (props) => {
+
+    let dropdownList = props.list.map((label) => {
```
Comment (sanghaiadarsh) [inline src/Components/Common/Dropdown/Dropdown.js:5]: use const

File: src/Components/Common/SearchBar/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { SearchIcon } from "../Icons";
+import React from "react";
+
+const SearchBar = (props) => {
+    let inputClasses = "h-12 focus:outline-none rounded py-2 block w-full appearance-none leading-normal ";
```
Comment (sanghaiadarsh) [inline src/Components/Common/SearchBar/SearchBar.js:5]: use const

File: src/Components/Containers/Modal.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        );
+    }
+    else {
+        enableScroll();
+        return ("");
+    }
+};
+
+const disableScroll = () => {
+    let name = "overflow-y-hidden";
```
Comment (sanghaiadarsh) [inline src/Components/Containers/Modal.js:42]: use const

File: src/Components/Containers/Modal.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    let arr = document.body.className.split(" ");
+    document.body.className += " " + name;
+};
+
+const enableScroll = () => {
+    let name = "overflow-y-hidden";
```
Comment (sanghaiadarsh) [inline src/Components/Containers/Modal.js:48]: use const

File: src/Components/Jobs/JobList.js [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        return (
+            <div className="flex-col w-full my-2">
+                <div className="flex mr-64">
+                    <Dropdown title="Sort By" label="Oldest" list={["Some", "Sample", "Data"]} />
+                    <div className="m-4" />
+                    <Dropdown title="Job status" label="Open" list={["Some", "Sample", "Data"]} />
+                </div>
+                <h2 className="text-base mt-6">Job Tags</h2>
+                {/* <p className="text-nebula-grey-500 mt-1 mb-2 text-sm">No tags added</p>
+                <SearchBar className="mt-2 mb-40 bg-nebula-grey-200 " inputClass="placeholder-nebula-grey-600 bg-nebula-grey-200" placeholder="Search for tags to add" /> */}
```
Comment (sanghaiadarsh) [inline src/Components/Jobs/JobList.js:65]: do we need the comments for future use?

File: src/Components/routes/CreateJob/CreateJob.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+                    <h2 className="text-base font-semibold">Difficulty</h2>
+                    <p className="text-nebula-grey-700 leading-tight text-sm">How difficult is the job?</p>
+                </div>
+                <Dropdown list={["Intermediate", "Easy", "Hard"]} />
+            </div>
+            <h2 className="text-base font-semibold mt-10">Skills required</h2>
+            {/* <InputFields className="mt-2 w-full mb-4" placeholder = "Type and press enter to add skills" />
+            <div className="flex flex-row flex-wrap">
+            <Tag label = "Nodejs" /> 
+            </div>*/}
```
Comment (sanghaiadarsh) [inline src/Components/routes/CreateJob/CreateJob.js:68]: remove the comments if they are not needed please.

**Review by sanghaiadarsh — CHANGES_REQUESTED** [review]
Left some minor comments

**Comment by Vulfpeck** [comment]
Will make the changes

**Pull request merged**